src/services/formatting/formatting.ts

```diff
@@ -863,7 +864,7 @@ function formatSpanWorker(
                 // if child node is a token, it does not impact indentation, proceed it using parent indentation scope rules
                 const tokenInfo = formattingScanner.readTokenInfo(child);
                 // JSX text shouldn't affect indenting
-                if (child.kind !== SyntaxKind.JsxText) {
+                if (child.kind !== SyntaxKind.JsxText && !isGrammarError(parent, child)) {

```

Comment:
Trying to understand the impact of this; if we format code that contains a grammar error node, will we now drop it?